nearclient.js

```diff
@@ -74,6 +75,7 @@ class NearClient {
             id: Date.now().toString(),
         };
         const response = await this.nearConnection.request('', request);
+        console.log('response', response);
```

Comment:
Also this

Reply:
I would actually like to have some sort of debug flag so that you can more easily have debug output from nearlib. (Can be a different PR)